### DISCUSSION TITLE
Баланс кошелька стал меньше

### DISCUSSION THREAD

**Discussion by Russsli** [Chia-Network/chia-blockchain#17659] [Support]
![image 1](https://github.com/Chia-Network/chia-blockchain/assets/161989397/fbfa5d64-f004-473a-8d3a-669490d296a4)
![image 2](https://github.com/Chia-Network/chia-blockchain/assets/161989397/6b5d937c-4c34-4344-ae2d-e2a84a7b1223)
My name is Ruslan. I kept my Chia's tokens on the official wallet - Chia Blockchain. I have two wallets: 478952959 - named Wallet and 3630580428 - named Wallet 2. When I tried to send tokens on the another wallet my ballance was changed and I lose 20 tokens in Wallet 478952959, and 7 tokens in Wallet 2 3630580428. I need your help. All of transactions you can see. My e-mail: <redacted>

 

 
 
 

 

 
![image 3](https://github.com/Chia-Network/chia-blockchain/assets/161989397/704b3124-993a-4f40-aba0-75d5f5f13064)
![image 4](https://github.com/Chia-Network/chia-blockchain/assets/161989397/aee4b032-2654-4ee4-9aac-3ad8b09254fa)

**Comment by BrandtH22** [comment]
Hey @Russsli , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

Unfortunately it appears someone has gained access to your keyset and/or device and stolen your funds. Since chia is a truly decentralized network there is no way to reclaim the funds or revert the transaction(s). One thing we recommend is ensuring that you and your farm are following security best practices to reduce malicious actors ability to gain access to your keys and/or device, many of these best practices are outlined here: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

The best place to reach our support team is on Discord (https://discord.gg/chia).

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
